src/components/table/table-row/classic-row.style.js

```diff
@@ -0,0 +1,49 @@
+import StyledTableCell from '../table-cell/table-cell.style';
+
+function applyClassicRowStyling() {
+  return `
+    border-color: #E7F1FC;
+    &:hover .common-input__input {
+      border-color: #E7F1FC;
+    }
+    &:nth-child(2n+1) {
+      ${StyledTableCell} {
+        background-color: #ffffff;
+      }
+    }
+    &:hover {
+      ${StyledTableCell} {
+        background-color: #E7F1FC;
+      }
+    }
+  `;
+}
+
+function applyClassicSelectedStyling(theme) {
+  const { colors } = theme;
```

Comment:
I'm not sure we need destructuring colors if we use  it only once

Reply:
Yeah,  good call